### ISSUE TITLE
Descriptions Not Showing For v1alpha1 Resources with list/describe Commands

### ISSUE THREAD

**Issue by danielhelfand** [tektoncd/cli#819]
# Version

```
Client version: <master branch>
Pipeline version: v0.11.0-rc2
```

# Expected Behavior

Using list/describe commands should show a description for `v1alpha1` resources.

# Actual Behavior

Description only works with `v1beta1` resources. 

# Steps to Reproduce the Problem

1. Create a `v1alpha` resource that supports description (Task, ClusterTask, Condition, Pipeline) and add a description property to the spec of the resource
2. Run `tkn <resource> ls` and `tkn <resource> desc <resourceName>`

**Comment by piyush-garg** [comment]
I am already on it

/aasign

**Comment by piyush-garg** [comment]
I think the issue is only with task and clustertask

**Comment by danielhelfand** [comment]
Yes, my apologies. I see that it works for pipelines/conditions.

**Comment by piyush-garg** [comment]
This will be resolved with https://github.com/tektoncd/pipeline/pull/2247 and we bump the dependency to v0.11-rc3 I will keep this open until we bump and will work toward bumping to rc3. Thanks

**Comment by piyush-garg** [comment]
This will be fixed #859
